packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/components/LogoModalStepper/FromUrlForm.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { Box } from '@strapi/design-system/Box';
+import { Button } from '@strapi/design-system/Button';
+import { TextInput } from '@strapi/design-system/TextInput';
+import { ModalFooter } from '@strapi/design-system/ModalLayout';
+import { urlToFile } from '../../utils/urlToFile';
+import { parseFileMetadatas } from '../../utils/parseFileMetadatas';
+import { SIZE, DIMENSION } from '../../utils/constants';
+
+const FromUrlForm = ({ goTo, next, onClose, setLocalImage }) => {
+  const { formatMessage } = useIntl();
+  const [logoUrl, setLogoUrl] = useState('');
```

Comment:
Can you use null instead of empty string?

Reply:
TextInput doesn't allow null value
<img width="291" alt="image" src="https://user-images.githubusercontent.com/71838159/163155591-51fa305c-55b8-4d32-a13d-62ac2de8cd62.png">
Since it's a controlled input, emprty string should be the way to go, WDYT?

Reply:
You are right :+1: